### ISSUE TITLE
README.md is out of date regarding logging settings

### ISSUE THREAD

**Issue by robertfw** [VSCodeVim/Vim#8529]
**Describe the bug**
v1.25.2 removed the `vim.debug.silent`, `vim.debug.loggingLevelForConsole`, and `vim.debug.loggingLevelForAlert` settings (see https://github.com/VSCodeVim/Vim/commit/f122b747a002610b9bf3df320e3270dffd712d68)

However, the README was not updated and still referred to these settings, both in the config reference and in the "Debugging Remappings" section.

**To Reproduce**
Read the Readme :wink: 

**Expected behavior**
Readme to be up to date :wink: 

I have a PR lined up to update accordingly

**Comment by j-steinbach** [comment]
Awesome, just had this problem that I didn't know how to debug my keys.

These are the correct steps (from robertfw's PR): 

```
   1.  Run `Developer: Set Log Level` from the command palette.
    2.  Select `Vim`, then `Debug`
    3.  Run `Developer: Reload window`
    4.  In the bottom panel, open the `Output` tab and select `Vim` from the dropdown selection.
```

**Comment by J-Fields** [comment]
Fixed by #8530
